src/bin/dump_sled.rs

```diff
@@ -0,0 +1,97 @@
+use std::env;
+use std::fs;
+use std::io::Write;
+use std::path::PathBuf;
+
+use anyhow::{Context, Result};
+use fnv::FnvHashMap;
+use rollup_state_manager::state::AccountState;
+use rollup_state_manager::test_utils::fr_to_string;
+use rollup_state_manager::types::merkle_tree::Tree;
+use rollup_state_manager::types::primitives::FrWrapper;
+
+fn main() -> Result<()> {
+    let sled_path: PathBuf = env::var("SLED_DB_PATH")
+        .unwrap_or_else(|_| "/tmp/rollup-sled.db".to_string())
+        .parse()?;
+    let dump_path: PathBuf = env::var("SLED_DUMP_PATH")
+        .unwrap_or_else(|_| "circuits/testdata/dump".to_string())
+        .parse()?;
+    fs::create_dir_all(&dump_path).context("Failed to create dump directory")?;
+
+    let db = sled::open(&sled_path).context("Failed to open sled")?;
+
+    let account_tree: Tree = db.get("account_tree")?.and_then(|v| bincode::deserialize(v.as_ref()).ok()).unwrap();
+    serde_json::to_writer_pretty(&mut fs::File::create(&dump_path.join("account_tree.json"))?, &account_tree)?;
+
+    let account_states = db.open_tree("account_states").unwrap();
```

Comment:
`account_states`, `balance_trees` and `order_trees` appear across different files (src/bin/dump_sled.rs and src/state/witness_generator.rs)

usuallly we prefer to use static/const for these _KEY_s so that we can avoid typos or name inconsistence due  to future renaming